content/build-cpi.md

```diff
@@ -29,7 +29,9 @@ The [`bosh_cpi`](https://rubygems.org/gems/bosh_cpi) gem provides a `Bosh::Cpi::
 
 There are a few CPI releases written in Go, as well:
 
+ * [Alibaba Cloud CPI Release](https://github.com/cloudfoundry-incubator/bosh-alicloud-cpi-release)
  * [Google CPI Release](https://github.com/cloudfoundry/bosh-google-cpi-release)
+ * [Google CPI Release](https://github.com/cloudfoundry-incubator/bosh-google-cpi-release)
```

Comment:
This seems unrelated to the PR. Is there a reason for adding this or was this a mistake?

Reply:
Mistake and fixed.